Integrations/integration-IntSights.yml

```diff
@@ -88,6 +89,67 @@ script:
                 return_error('Error in API call to IntSights service %s - [%d] %s' % (path, r.status_code, r.text))
         return r
 
+    def convert_iso_string_to_python_date(date_in_iso_format):
+        iso_format = "%Y-%m-%dT%H:%M:%S"
+        date_in_python_format = datetime.strptime(date_in_iso_format, iso_format)
+        return date_in_python_format
+
+    def convert_python_date_to_unix_millisecond(python_date_object):
```

Comment:
Take a look at the script CommonServerPython located in the Automations section in your Demisto instance